Combobox/Combobox.html

```diff
@@ -1,6 +1,5 @@
-<template class="d-combobox" role="presentation">
+<template class="d-combobox" role="combobox" aria-expanded="{{this.opened}}" attach-point="popupNode">
```

Comment:
You can (and should) actually just do `{{opened}}` rather than `{{this.opened}}`.